packages/core/admin/admin/src/content-manager/pages/ListSettingsView/index.jsx

```diff
@@ -83,7 +82,7 @@ export const ListSettingsView = ({ layout, slug }) => {
     {
       onSuccess() {
         trackUsage('didEditListSettings');
-        refetchData();
+        queryClient.invalidateQueries(['contentManager', 'init']);
```

Comment:
I would export this key and import it here so we're _certain_ to use the same one and avoid silly spelling errors since i don't believe theres any type safety on query keys?